jmxtrans-output/jmxtrans-output-elastic/src/main/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriter.java

```diff
@@ -128,7 +128,7 @@ protected void internalWrite(Server server, Query query, ImmutableList<Result> r
 					map.put("key", values.getKey());
 					map.put("keyAlias", result.getKeyAlias());
 					map.put("value", Double.parseDouble(value.toString()));
-					map.put("timestamp", result.getEpoch());
+					map.put("timestamp", java.util.Date(result.getEpoch()));
```

Comment:
![image](https://cloud.githubusercontent.com/assets/9464485/13598570/78abeb4c-e559-11e5-80ba-307284d14fe6.png)
this is ok